notebooks/303-pytorch-int8-semseg-with-nncf-pot-openvino/303-pytorch-int8-semseg-with-nncf-pot-openvino.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "af7b6f13",
+   "metadata": {},
+   "source": [
+    "# INT8 acceleration of semantic segmentation with NNCF/POT for OpenVINO\n",
+    "\n",
+    "This notebook runs through the process of enabling [NNCF](https://github.com/openvinotoolkit/nncf) in a simple semantic segmentation pipeline for training UNet on Camvid, and comparing results against the Post-Training Optimization Tool ([POT](https://docs.openvinotoolkit.org/latest/pot_README.html))'s quantization.\n",
+    "\n",
+    "> NOTE: _For this notebook to function, please make sure that your Python environment has `openvino`, `openvino-dev` and `nncf[torch]` packages installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27cb5c08",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49c29922",
+   "metadata": {},
+   "source": [
+    "Get the dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2a33c1af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/alexgkendall/SegNet-Tutorial data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24d32df7",
+   "metadata": {},
+   "source": [
+    "### Obtaining the uncompressed (FP32) performance and accuracy baselines\n",
+    "Download the FP32 pre-trained weights checkpoint for UNet on CamVid:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4ad25d7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://storage.openvinotoolkit.org/repositories/nncf/openvino_ckpts/303_unet_camvid.pth -O unet_camvid.pth"
```

Comment:
Done

Reply:
Sevaral questions/suggestions to @vshampor :
- Do we really need to clone a 3rd party repo in this notebook? We have a segmentation sample in the NNCF. Can we use it?
- Some additional information should be added to the notebook to make it more informative, like "This tutorial shows advantages of Quantization-Aware Training available in the NNCF vs. Post-training Quantization methods ...". We need to highlight what is going to be demonstrated and what are the benefits of NNCF. Now it is not clear.

Reply:
@AlexKoff88 in fact what we have here with respect to the training pipeling is based on the sample we have in NNCF. The version of the training pipeline you see here is refactored so that it co-locates the changes required for NNCF integration -  I did so in order to make the integration process is clear and concise for this specific notebook. 

As for the second note - don't we have all of that in the neighbouring NNCF-powered notebooks?

Reply:
@vshampor , in my opinion each notebook should maximally independent from other notebooks.

Reply:
Regarding the last comment, I agree with Maxim that each notebook should be self-sufficient so that the newcomers don't need to go and learn about NNCF in the neighboring notebooks. 
As for the sample, I don't think that the availability of the patch makes the integration process clear. We could have a fork of that repo and it would look the same. Actually, I would like to clarify the goal of this tutorial. To me it looks now that the main goal of it is to demonstrate the advantages of QAT vs. PQT. If this it so, the NNCF  built-in segmentation sample can also suite for this purpose.